images/release/Dockerfile

```diff
@@ -15,13 +15,17 @@ ENV VERSION=1.6 \
 ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
 
 RUN mkdir $TMPDIR && \
-    INSTALL_PKGS="make gcc zip mercurial krb5-devel bsdtar bc rsync bind-utils file jq openssl" && \
+    INSTALL_PKGS="make gcc zip mercurial krb5-devel bsdtar bc rsync bind-utils file jq tito openssl" && \
     yum install -y $INSTALL_PKGS && \
     rpm -V $INSTALL_PKGS && \
     yum clean all && \
     curl https://storage.googleapis.com/golang/go$VERSION.linux-amd64.tar.gz | tar -C /usr/local -xzf - && \
     go get golang.org/x/tools/cmd/cover github.com/tools/godep github.com/golang/lint/golint && \
-    touch /os-build-image
+    git clone https://github.com/dgoodwin/tito.git && \
+    pushd tito && yum-builddep -y tito.spec && tito build --test --rpm && rpm --upgrade --force /tmp/tito/noarch/tito*.rpm && popd && \
```

Comment:
How long does this take?  Also, remove the tito directory after installing the RPM